app/assets/stylesheets/_variables.scss

```diff
@@ -18,37 +12,23 @@ $header-line-height: 1.25;
 // Other Sizes
 $base-border-radius: 3px;
 $base-spacing: $base-line-height * 1em;
+$small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477DCA;
+$blue: #477dca;
 $dark-gray: #333;
 $medium-gray: #999;
-$light-gray: #DDD;
-
-// Background Color
-$base-background-color: white;
```

Comment:
It's not just insets. I use it for plenty of things such as inverted texts and subtle shadings and inverted areas. I vote for keeping it.